### PR TITLE
Instance Electronic Access Metadb derived

### PR DESCRIPTION
**Pull request by jenmawe**
Porting over instance electronic access derived table from LDP to Metadb. No issue related to this pull request.

### CODE REVIEW

File: sql_metadb/derived_tables/instance_electronic_access.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+)
+SELECT 
+	ea.instance_id,
+	ea.instance_hrid,
+	ea.uri,
+	ea.link_text,
+	ea.materials_specification,
+	ea.public_note,
+	ea.relationship_id,
+	eart."name" AS relationship_name
```
Comment (adoerrer) [inline sql_metadb/derived_tables/instance_electronic_access.sql:30]: The ordinality has been extracted in the sub query but has not brought to the main query.

**Review by adoerrer — CHANGES_REQUESTED** [review]
Beside the missing ordinality in the main query the query works great, thank you!

File: sql_metadb/derived_tables/instance_electronic_access.sql (hunk 1 above)
Comment (jenmawe) [reply sql_metadb/derived_tables/instance_electronic_access.sql]: Thanks for catching that Axel. I made the changes and fixed the other things that have come up in other reviews.

**Review by adoerrer — APPROVED** [review]
Looks good and works great, thank you!

**Review by amzoss — APPROVED** [review]
Thanks, looks good!

**Pull request merged**